pkg/logging/splunk/update.go

```diff
@@ -17,7 +17,7 @@ type UpdateCommand struct {
 	manifest manifest.Data
 
 	// required
-	EndpointName string // Can't shaddow common.Base method Name().
```

Comment:
@kellymclaughlin looks like you missed out one instance:

```
// UpdateCommand calls the Fastly API to update Splunk logging endpoints.
```

Did you leave this in just as a way to ensure I reviewed every single file? 😉 well... I did check _every single file_ lol 😆

Reply:
Darn it. How did I miss that? :facepalm:

Reply:
Fixed